ReadMe.md

```diff
@@ -148,6 +148,10 @@ You can now run the various Run/Debug Configurations such as
 * All Compiler Tests
 * All IDEA Plugin Tests
 
+### Running specific generated tests
+
+If you need to debug a specific generated test ensure that you have the `Working directory` your IntellJ run configuration set
```

Comment:
> generated test ensure that

* generated test`,` ensure that

> you have the \`Working directory\` your IntellJ

* you have the \`Working directory\` `in` your Intell`i`J

> If you don't every test

If you don't`,` every test

Reply:
Thanks, will fix those. I knew what I had written didn't sound quite right. COMMAS!